content/rc/administration/setup/customize-flexible-plan.md

```diff
@@ -0,0 +1,105 @@
+---
+Title: Customize a Flexible plan
+description:
+weight: 30
+alwaysopen: false
+categories: ["RC"]
+aliases: /rc/administration/customize-pro/
+---
+
+When you create a subscription with a Flexible plan, you customize it to your business needs.
+
+This means you:
```

Comment:
I love that you layout the main steps of the process. Puts the rest into perspective for the reader!